docs/kqueen.cli_gcloud_examples.rst

```diff
@@ -0,0 +1,137 @@
+Kqueen and Google cloud from scratch
+------------------------------------
+
+Create the Google Cloud account and start with `gcloud`:
+
+https://cloud.google.com/pubsub/docs/quickstart-cli
+
+Login to Google Cloud
+
+.. code-block:: bash
+
+   $ cd /tmp/
+   $ gcloud init
+
+Create new project in GCE
+
+.. code-block:: bash
+
+   $ GCE_PROJECT_NAME="My Test Project"
+   $ GCE_PROJECT_ID="my-test-project-`date +%F`"
+   $ gcloud projects create "$GCE_PROJECT_ID" --name "$GCE_PROJECT_NAME"
+   Create in progress for [https://cloudresourcemanager.googleapis.com/v1/projects/my-test-project-2018-05-11].
+   Waiting for [operations/cp.7874714971502871878] to finish...done.
```

Comment:
remove log output

Reply:
and $

Reply:
I prefer to keep the outputs wherever is possible - to see real outputs + real examples. 
This helps users to ensure they are getting the right "similar" output (and it should not harm anybody)...

Let me know if you really want to remove all the outputs...